.gitignore

```diff
@@ -40,4 +40,6 @@ cmake_install.cmake
 
 Roy_the_Traffic_Cop
 Roy_the_Traffic_Cop.cbp
+/visual_studio/tmp/
+/visual_studio/Debug/
```

Comment:
I think in addition to adding these files to the gitignore, you have to do `git rm -r -f foldername` to actually remove the files. Since they've already been committed, git will keep tracking them if you don't.

Reply:
Will do!